app/src/service/ahgoraService.js

```diff
@@ -1,129 +1,123 @@
+    console.log(comand)
+    window.Neutralino.os.execCommand(comand)
   },
   espelhoPonto: function (year, month) {
-    let today = new Date();
-    let mirroDate = new Date(year,parseInt(month)-1,1);
+    const today = new Date()
+    const mirroDate = new Date(year, parseInt(month) - 1, 1)
 
     return new Promise((resolve, reject) => {
-      let jwt = JSON.parse(localStorage.getItem("userDetails")).jwt;
+      const jwt = JSON.parse(localStorage.getItem('userDetails')).jwt
 
-      const header = `cookie: qwert-external=${jwt}`;
-      let comand = `curl -H "${header}" -X GET https://www.ahgora.com.br/api-espelho/apuracao/${year}-${month}`;
+      const header = `cookie: qwert-external=${jwt}`
+      const comand = `curl -H "${header}" -X GET https://www.ahgora.com.br/api-espelho/apuracao/${year}-${month}`
 
       window.Neutralino.os.execCommand(comand).then((result) => {
-        let mirror = JSON.parse(result.stdOut);
-        console.log(mirror);
+        const mirror = JSON.parse(result.stdOut)
+        console.log(mirror)
         if (mirror.error) {
-          reject(mirror);
+          reject(mirror)
         } else {
-          localStorage.setItem("mirror", JSON.stringify(mirror));
-          resolve(mirror);
+          localStorage.setItem('mirror', JSON.stringify(mirror))
+          resolve(mirror)
 
-          var obj = {
+          const obj = {
             historico: {
-                referencia:{
-                  [year+'-'+month]: {
-                    dias: mirror.dias,
-                    total: mirror.meses[year+'-'+month]
-                  }
+              referencia: {
+                [year + '-' + month]: {
+                  dias: mirror.dias,
+                  total: mirror.meses[year + '-' + month]
                 }
-              },
+              }
+            }
           }
 
-          if( mirroDate.getMonth() < today.getMonth()){
-            window.Neutralino.storage.setData('history', JSON.stringify(obj));
+          if (mirroDate.getMonth() < today.getMonth()) {
+            window.Neutralino.storage.setData('history', JSON.stringify(obj))
           }
         }
-      });
-    });
+      })
+    })
   },
   baterPonto: function () {
     return new Promise((resolve, reject) => {
-
-      let user = JSON.parse(localStorage.getItem("credential"));
-      const identity = JSON.parse(localStorage.getItem("identity"));
+      const user = JSON.parse(localStorage.getItem('credential'))
+      const identity = JSON.parse(localStorage.getItem('identity'))
       const credential = new URLSearchParams({
         identity: identity.identity,
         account: user.username,
         password: user.password,
-        origin: "pw2",
-      });
+        origin: 'pw2'
+      })
 
-      let comand = `curl -d "${credential}" -X POST http://www.ahgora.com.br/batidaonline/verifyIdentification`;
+      const comand = `curl -d "${credential}" -X POST http://www.ahgora.com.br/batidaonline/verifyIdentification`
```

Comment:
Codacy has a fix for the issue: [Insert `;`](https://app.codacy.com/gh/ruizivo/ahgora-login/pullRequest?prid=9873926)
```suggestion
      const comand = `curl -d "${credential}" -X POST http://www.ahgora.com.br/batidaonline/verifyIdentification`;
```